tensorflow/compiler/mlir/tosa/transforms/convert_tfl_uint8.cc

```diff
@@ -130,6 +130,63 @@ struct ConvertUint8QConstOp : public RewritePattern {
   }
 };
 
+namespace {
+
+// returns true iff @a type is a shaped type with element type that is uint8
+// if it is, then return the rescaled type, uint8_zp, and output_zp to use to
+// rescale type to signed type with adjusted zero point.
+bool IsShapedUint8Type(OpBuilder& builder, const Type type, Type& rescaled_type,
+                       int32_t& uint8_zp, int32_t& output_zp) {
+  auto uint8_type = type.dyn_cast<mlir::ShapedType>();
```

Comment:
This name is not accurate. It would be called something like ``shaped_type`.

Reply:
changed to getUint8RescaleInfo